components/apimgt/org.wso2.carbon.apimgt.keymgt/src/main/java/org/wso2/carbon/apimgt/keymgt/ScopesIssuer.java

```diff
@@ -37,18 +40,37 @@
     private List<String> scopeSkipList = new ArrayList<>();
     private static Map<String, AbstractScopesIssuer> scopesIssuers;
     private static final String DEFAULT_SCOPE_NAME = "default";
+    private static final String CONFIG_ELEM_SCOPE_ISSUER = "OAuthConfigurations.ScopeIssuer";
     /**
      * Singleton of ScopeIssuer.*
      */
     private static ScopesIssuer scopesIssuer;
 
-    private ScopesIssuer() {
+    public ScopesIssuer() {
 
     }
 
-    public static void loadInstance(List<String> whitelist) {
-
-        scopesIssuer = new ScopesIssuer();
+    public static void loadInstance(List<String> whitelist) throws APIKeyMgtException{
+        APIManagerConfiguration configParser = ServiceReferenceHolder.getInstance().getAPIManagerConfigurationService()
```

Comment:
make configuration

Reply:
fixed in 0f79f06894f367c51340a467bfb987559791776c